xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1038,6 +1040,9 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
   if (pThumbs)
     GetCustomExtensions(pThumbs,m_musicThumbs);
 
+  // show art for shoutcast v2 streams (set to false for devices with limited storage)
+  XMLUtils::GetBoolean(pRootElement, "showshoutcastart", m_bShowShoutcastArt);
```

Comment:
Please update the [wiki](https://kodi.wiki/view/Advancedsettings.xml) for `showshoutcastart`

Reply:
Yep, I'll do that once its merged.